src/components/Topbar.vue

```diff
@@ -41,13 +41,17 @@ export default {
             return startTime + this.duration * 1000
         },
         formattedTimeRemaining() {
-            const minutes = Math.floor(this.timeRemaining / 60)
+            const hours = Math.floor(this.timeRemaining / 3600)
+            const minutes = Math.floor((this.timeRemaining % 3600) / 60)
             let seconds = this.timeRemaining % 60
 
             if (seconds < 10) {
                 seconds = `0${seconds}`
             }
```

Comment:
might wanna do the same for hours and mins as well, just to be consistent

Reply:
:+1: